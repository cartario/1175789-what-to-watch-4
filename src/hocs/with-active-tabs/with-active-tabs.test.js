import React from "react";
import renderer from "react-test-renderer";
import withActiveTabs from "./with-active-tabs.js";
import PropTypes from "prop-types";

const MockComponent = (props) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

MockComponent.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired,
};

const MockComponentWrapped = withActiveTabs(MockComponent);

describe(`withActiveTabs Test`, () => {
  it(`should render withActiveTabs`, () => {
    const tree = renderer.create((
      <MockComponentWrapped

      />
    ), {
      createNodeMock() {
        return {};
      }
    }
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
