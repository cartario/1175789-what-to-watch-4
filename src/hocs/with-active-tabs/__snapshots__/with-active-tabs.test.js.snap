// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveTabs Test should render withActiveTabs 1`] = `<div />`;
